__tests__/app.test.js

```diff
@@ -113,5 +113,39 @@ describe("get /api/articles", () => {
     });
 });
 
+describe("get /api/articles/:article_id/comments", () => {
+    test("returns an article object by id with appropriate content keys", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.comments).toHaveLength(11);
+                body.comments.forEach(comment => {
+                    expect(comment.article_id).toBe(1);
+                    expect(typeof comment.comment_id).toBe('number');
+                    expect(typeof comment.votes).toBe('number');
+                    expect(typeof comment.created_at).toBe('string');
+                    expect(typeof comment.author).toBe('string');
+                    expect(typeof comment.body).toBe('string');
+                })
+            });
+    });
+    test("returns a status 404 error message when a valid article id without comments is searched", () => {
```

Comment:
This should be a 200, nothing has gone "wrong" as such so there is no need for an error, we'd just want to see an empty array

Reply:
You are also missing a 404 test for non existent articles Ie /api/articles/99999/comments 

You will need a checkExists function to differentiate between 200 - existing articles with no comments, and 404 - article doesn't exist. _both_ of these will return empty rows from the database []